.github/workflows/file_health_exclusions_nonutf8.txt

```diff
@@ -0,0 +1,85 @@
+./dist/mac/CMakeLists.txt
```

Comment:
Why is it non-UTF8?

Reply:
@glassez 

Because it's empty. With the current method (`file --mime`), empty files can't be considered UTF-8, regardless of their purpose or extension:

```
$ file --mime dist/mac/CMakeLists.txt 
dist/mac/CMakeLists.txt: inode/x-empty; charset=binary
```

Reply:
fixed.